misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -409,12 +418,18 @@ public ReturnTypeDescriptorNode transform(ReturnTypeDescriptorNode returnTypeDes
         if (!isInLineRange(returnTypeDescriptorNode, lineRange)) {
             return returnTypeDescriptorNode;
         }
+        boolean addSpaces = false;
+        if (returnTypeDescriptorNode.parent() != null &&
+                returnTypeDescriptorNode.parent().kind() == SyntaxKind.BLOCK_STATEMENT) {
+            addSpaces = true;
+        }
```

Comment:
Can simplify the addSpaces variable assignment

Reply:
Will make the following improvement in a refactoring PR.
`boolean addSpaces = returnTypeDescriptorNode.parent() != null &&
                returnTypeDescriptorNode.parent().kind() == SyntaxKind.BLOCK_STATEMENT;`